app/uk/gov/hmrc/apidocumentation/views/fraudPrevention.scala.html

```diff
@@ -26,7 +26,7 @@
   <div class="fraud-prevention">
```

Comment:
Just to make sure, we still want to keep (line 415)

> If you cannot discover any information due to operating system constraints, leave the appropriate field blank as shown in the following examples:

Reply:
Yes